src/main/java/Deadline.java

```diff
@@ -0,0 +1,13 @@
+public class Deadline extends Task {
+    protected String by;
```

Comment:
I think a more descriptive variable name would be more helpful.
```suggestion
    protected String dueDate;
```